### DISCUSSION TITLE
Molecule concentration in MoBi is not zero also if the molecule is not expressed

### DISCUSSION THREAD

**Discussion by teutonicod** [Open-Systems-Pharmacology/Forum#1631] [Questions & Problems]
Dear all,

I am having an issue with MoBi V11 which I am not able to solve.
I needed to create a tumor compartment in MoBi with a target expressed. 
These are the steps I followed:
-I created the target in PK-Sim and expressed it in the muscle
-I exported the model in MoBi and cloned the muscle to create the tumor compartment
-I follow the usual steps to create a tumor compartment. The simulation runs correctly.
-I then wanted to take away the target in the muscle and expressed it in the tumor. I first changed the relative expression and set it to 1 in the tumor. Now the target concentrations appeared in the tumor and in the muscle. Then I set up the relative expression in the muscle to 0 but in the simulation, I still see the target appearing.

The building blocks are updated, the simulation as the right reference concentration (0) and I also tried to configure again the simulation but still the same problem.

Could you tell me if is there any other parameter that could explain why the target still appears in the muscle?

Thanks in advance for your help!

Best,

Donato

**Comment by Christoph27** [comment]
Hi Donato,

I recently added a tumor with in V11 and it worked without problems (also regarding expression on the organ which was cloned). 

Maybe you set "Relative expression in vascular endothelium" > 1? Where is your target supposed to be localized within tissue?

Best, Christoph

**Reply by teutonicod** [reply]
Hi Christoph,

The "relative expression in vascular endothelium" is set to 0. The target is supposed to be on the surface of cells, so I have a "relative expression" of 1 intracellular and a "fraction expressed in intracellular" set to 0.

Actually, in my case, I had some issues with expression, since the parameter start values for the target were not created in the tumor when I expanded the BB. I had to create them manually copying the one from the muscle. I don't know if this could be a problem, but it seems to work since I see the target in the tumor. I just cannot get rid of the one in the muscle.

Best,

Donato

**Comment by Yuri05** [comment]
@teutonicod 
hard to say without the project. Can you upload it or send me per email?

**Reply by teutonicod** [reply]
Hi Juri,

Thanks a lot for your reply. I just found the issue. The target was also expressed in plasma and it was appearing from there, even if it was not expressed in the muscle. Thanks a lot to you and @Christoph27 for your help!

Donato
